src/components/EditorBar/EditorBar.styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { colors } from '@keen.io/colors';
+
+export const Container = styled.div`
+  height: 47px;
+  width: 100%;
+  background-color: ${colors.white[500]};
+  box-shadow: 0 10px 24px 0 rgba(29, 39, 41, 0.15);
+  div.right {
+    display: inline-block;
+    float: right;
```

Comment:
can we use `flex` instead of `floats`? in this case we don't need to clear the floats